### PR TITLE
Dev/profile

### PR DESCRIPTION
**Pull request by shivamgoyal7**
### Functionality:
added profile page

### Solution:
added profile page and edit option to update user data and updated theme functionality

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators

Untitled

### CODE REVIEW

File: functions/createNewUserAPI.js [hunk 1]
```diff
@@ -44,6 +44,8 @@ exports.createNewUser = functions.https.onRequest((request, response) => {
                     email: Email,
                     phoneNumber: PhoneNumber,
                     providerId: ProviderId,
+                    Bio: "",
```
Comment (viv9k) [inline functions/createNewUserAPI.js:47]: "About Me : " would sound better than bio

File: functions/updateUserProfileAPI.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        db.collection("Users").doc(uid).update({
+            displayName: displayName,
+            Bio: bio,
+        }).then(() => {
+            result = { data: "User Profile updated successfully" };
+            console.log("Successful");
+            return response.status(200).send(result);
+        }).catch((error) => {
+            result = { data: error };
+            console.error("Error", error);
```
Comment (viv9k) [inline functions/updateUserProfileAPI.js:30]: Can we merge this into addUsers api

File: src/app/Interface/UserInterface.ts [hunk 3]
```diff
@@ -10,4 +10,5 @@ export interface User {
 export interface UserAppSetting extends User {
     AppKey: string;
     TeamId: string;
+    Bio: string;
```
Comment (viv9k) [inline src/app/Interface/UserInterface.ts:13]: Bio --> About Me

File: src/app/body/profile/edit-profile/edit-profile.component.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                    </div>
+                    <div class="form-group">
+                        <label for="message-text" class="col-form-label">Bio:</label>
+                        <textarea class="form-control" [(ngModel)]="bio" [ngModelOptions]="{standalone: true}"
+                            id="message-text"></textarea>
+                    </div>
+                </form>
+                <div class="row px-4 py-4">
+                    <button type="button" class="ml-auto col-md-3 btn" data-dismiss="modal" aria-label="Close"
+                        (click)="editProfile()">Save</button>
```
Comment (viv9k) [inline src/app/body/profile/edit-profile/edit-profile.component.html:26]: Better to use material icon

File: src/app/services/auth.service.ts [hunk 5]
```diff
@@ -88,5 +88,9 @@ export class AuthService {
   getTeamId() {
     return this.userAppSetting.TeamId;
   }
+  
+  getBio() {
+    return this.userAppSetting.Bio;
+  }
```
Comment (viv9k) [inline src/app/services/auth.service.ts:94]: Better to use About me

**Review by viv9k — CHANGES_REQUESTED** [review]
Please complete the required changes and show me before submitting it.

**Comment by viv9k** [comment]
Let's take quick action on this for tomorrows release.

File: functions/createNewUserAPI.js (hunk 1 above)
Comment (shivamgoyal7) [reply functions/createNewUserAPI.js]: ok

File: functions/updateUserProfileAPI.js (hunk 2 above)
Comment (shivamgoyal7) [reply functions/updateUserProfileAPI.js]: ok

**Review by viv9k — APPROVED** [review]
Ship it

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**